http/security/authenticator.go

```diff
@@ -0,0 +1,18 @@
+// Copyright © 2019 by PACE Telematics GmbH. All rights reserved.
+// Created at 2019/10/31 by Charlotte Pröller
+
+package security
+
+import (
+	"context"
+	"net/http"
+)
+
+// Authenticator describes the needed functions for a authentication,
+// already implemented in oauth2.Authenticator and apikey.Authenticator
+type Authenticator interface {
```

Comment:
If it's only method is `Authorize`, shouldn't the type actually be called `Authorizer` instead of `Authenticator`?